modules/api/src/main/GameApiV2.scala

```diff
@@ -100,6 +102,7 @@ final class GameApiV2(
       .mapConcat(_ filter config.postFilter)
       .take(config.max | Int.MaxValue)
       .via(preparationFlow(config))
+      .merge(Source.tick(keepAliveInterval, keepAliveInterval, emptyMsgFor(config)))
```

Comment:
maybe better: https://doc.akka.io/docs/akka/current/stream/operators/Source-or-Flow/keepAlive.html

Reply:
Definitely better. Should I also change the other place we use this?